src/manager/lotus.rs

```diff
@@ -39,11 +74,33 @@ impl SubnetManager for LotusSubnetManager {
         panic!("not implemented")
     }
 
-    async fn submit_checkpoint(&self, _subnet: SubnetID, _from: Address, _ch: Checkpoint) -> Result<()> {
+    async fn submit_checkpoint(
+        &self,
+        _subnet: SubnetID,
+        _from: Address,
+        _ch: Checkpoint,
+    ) -> Result<()> {
         panic!("not implemented")
     }
 
     async fn list_child_subnets(&self, _subnet: SubnetID) -> Result<HashMap<SubnetID, SubnetInfo>> {
         panic!("not implemented")
     }
 }
+
+impl<T: JsonRpcClient + Send + Sync> LotusSubnetManager<T> {
+    async fn get_subnet_actor_code_cid(&self) -> Result<Cid> {
+        let network_version = self.lotus_client.state_network_version(vec![]).await?;
+        self.state_actor_code_cids(network_version).await
+    }
+
+    async fn state_actor_code_cids(&self, network_version: NetworkVersion) -> Result<Cid> {
```

Comment:
You are returning the code cid for the subnet actor gere, not the whole map. I would suggest renaming the function and adding a description in a comment. 
```suggestion
    async fn subnet_actor_code_cid(&self, network_version: NetworkVersion) -> Result<Cid> {
```

Reply:
Updated and resolving this.